### PR TITLE
updates to data types

### PR DESCRIPTION
**Pull request by kitaf004**
Pull to update data types for ASMT fields

### CODE REVIEW

**Comment by kitaf004** [comment]
Still tweaking - will update when done

**Comment by kitaf004** [comment]
Ready for review

File: NCATS_Data_Dictionary.md [hunk 1]
```diff
@@ -808,93 +808,97 @@ document for [trustymail](https://github.com/cisagov/trustymail).
...
-- `Fed Lead`:  Federal Team Lead assigned to the assessment
-- `Contractor Count`:  Number of contractors assigned
-- `Draft w/ POC Date`:  Date when draft report is sent to the customer
-- `Fed Count`:  Number of Federal operators assigned
-- `Report Final Date`:  Date when report is marked Final
-- `Operator`:  Name of the Operator (Contractor or Fed)
-- `Stakeholder Id`:  TBD
-- `Testing Begin Date`:  Date when all testing begins
+- `Completed` [ISO date]: Date when assessment was completed
+- `Contractor Count` [interger]:  Number of contractors assigned
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: Here and in several places below, you misspell `integer` as `interger`

File: NCATS_Data_Dictionary.md [hunk 2]
```diff
@@ -808,93 +808,97 @@ document for [trustymail](https://github.com/cisagov/trustymail).
...
-- `POC Email`:  POC E-mail address
-- `POC Name`:  POC Name
-- `POC Phone`:  POC Phone Number
-- `ROE Date`:  Date ROE is signed
-- `ROE Number`:  ROE Number (assigned by NCATS)
-- `ROE Signed`:  ROE Signed (Boolean)
-- `_ID`:  Assessment ID (RV0XXX for RVA/HVA/PCA or VR0XXX for VADR)
-- `Asmt Name`:  Assessment Name (Usually customer name and assessment
+- `_ID` [ObjectId]:  Assessment ID (RV0XXX for RVA/HVA/PCA or VR0XXX for VADR)
+- `App A Date` [ISO date]:  Date Appendix A was signed
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: There is inconsistent capitalization of "Appendix" in this section.  Pick either `Appendix` or `appendix` and update accordingly.  @kitaf004, I know this is beyond the scope of what you were changing, but it caught my eye and should be fixed.

File: NCATS_Data_Dictionary.md [hunk 3]
```diff
@@ -808,93 +808,97 @@ document for [trustymail](https://github.com/cisagov/trustymail).
...
-- `Requested Svcs`:  Array containing list of NCATS services requested
-  for this engagement (List of Services here)
-- `Stakeholder Name`:  Stakeholder Name
-- `State` :  State where stakeholder is located
-- `Testing Complete Date`:  Date on which all testing is completed
-- `Testing Phase`:  Phase of Testing (External / Internal)
-- `Election`:  Elections Related (Boolean)
-- `Sector`:  Fed/State/Local/Tribal/Territorial/Critical Infrastructure
-- `CI Type`:  Critical Infrastructure Type (Selected from among 16 CI
+- `Asmt Type` [string]:Assessment Type (RVA, HVA, RPT, PCA, VADR)
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: Missing spaces after the colon

File: NCATS_Data_Dictionary.md [hunk 4]
```diff
@@ -52,82 +52,82 @@ This information is organized by database and collection (table).
 The data in this collection is derived from the National Vulnerability
 Database [CVE feeds](https://nvd.nist.gov/vuln/data-feeds).
 
-- `_id` [string]: [Common Vulnerabilities and
+- `_id` [string]:[Common Vulnerabilities and
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: Somehow you managed to remove all of the spaces after the colons in the upper portion of the data dictionary (everything outside of the `assessment` database).  Please return our missing spaces and no further questions will be asked.

Comment (kitaf004) [reply NCATS_Data_Dictionary.md]: They're being held in an undisclosed location until our demands are met.

File: NCATS_Data_Dictionary.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
 ## March 28, 2019 ##
 
 This document provides a data dictionary for the data stored in the
-following NoSQL MongoDB databases:
+following NoSQL MongoDB databases:
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: Your rampaging "search and replace all" is putting spaces after colons where no spaces should be, like here.

File: NCATS_Data_Dictionary.md [hunk 6]
```diff
@@ -13,12 +13,12 @@ following NoSQL MongoDB databases:
   * HTTPS web server scanning
   * SSL server scanning
   -  Certificates and pre-certificates from [Certificate
-     Transparency](https://www.certificate-transparency.org) logs
+     Transparency](https: //www.certificate-transparency.org) logs
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: And here.  And every other URL in this markdown.

File: NCATS_Data_Dictionary.md [hunk 7]
```diff
@@ -13,12 +13,12 @@ following NoSQL MongoDB databases:
...
 - `assessment` - Risk/Vulnerability Assessment (RVA) management data
 
 This information is organized by database and collection (table).
 
-[Cyhy Database:](#cyhy-database)
+[Cyhy Database: ](#cyhy-database)
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: And also here and in our other database headers.

File: NCATS_Data_Dictionary.md [hunk 8]
```diff
@@ -100,14 +100,14 @@ CyHy stakeholders.
...
-  - -4: Most severe vulnerability detected on this host is Medium
+  - -8:Most severe vulnerability detected on this host is High severity
+  - -4:Most severe vulnerability detected on this host is Medium
     severity
-  - -2: Most severe vulnerability detected on this host is Low severity
-  - -1: No vulnerabilities detected on this host
-  -  1: Host document represents a "dark space" IP address; i.e. live
+  - -2:Most severe vulnerability detected on this host is Low severity
+  - -1:No vulnerabilities detected on this host
+  -  1:Host document represents a "dark space" IP address; i.e. live
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: The colons in this section should still have spaces after them.

File: NCATS_Data_Dictionary.md [hunk 9]
```diff
@@ -473,51 +473,51 @@ CyHy stakeholders.
 
 The data in this collection is derived from certificates collected by
 our [Certificate
-Transparency](https: //www.certificate-transparency.org/) log scanner,
+Transparency](https://www.certificate-transparency.org/) log scanner,
 which only grabs certificates that apply to domains in our [domains
-collection](#h.qwt153c8i5hu).  NOTE: More details may be available in
+collection](#h.qwt153c8i5hu).  NOTE:More details may be available in
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: Places where we have `NOTE:` should have a space after the colon (there are a few more of these below).

Also, I just noticed that this link to the domains collection is messed up; not sure when it happened, but can you please fix it?  It should look like this:
`[domains collection](domains-collection)`

Comment (kitaf004) [reply NCATS_Data_Dictionary.md]: Changed the other items, but confused on domains collection link. I'm showing this (is this right?):
[domains Collection](#domains-collection)

File: NCATS_Data_Dictionary.md [hunk 10]
```diff
@@ -475,7 +475,7 @@ The data in this collection is derived from certificates collected by
 our [Certificate
 Transparency](https://www.certificate-transparency.org/) log scanner,
 which only grabs certificates that apply to domains in our [domains
-collection](#h.qwt153c8i5hu).  NOTE:More details may be available in
+collection](#h.qwt153c8i5hu).  NOTE: More details may be available in
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: This link wasn't updated, despite your bold claim to the contrary.

File: NCATS_Data_Dictionary.md [hunk 11]
```diff
@@ -100,10 +100,10 @@ CyHy stakeholders.
   database
 - `priority` [integer]: Scan priority of this host document, from -16
   (most urgent) to 1 (least urgent)
-  - -16: Most severe vulnerability detected on this host is Critical
+  - -16:Most severe vulnerability detected on this host is Critical
     severity
-  - -8: Most severe vulnerability detected on this host is High severity
-  - -4: Most severe vulnerability detected on this host is Medium
+  - -8:Most severe vulnerability detected on this host is High severity
+  - -4:Most severe vulnerability detected on this host is Medium
```
Comment (dav3r) [inline NCATS_Data_Dictionary.md]: You fixed some of the items in this section, but the first three (`-16`, `-8`, `-4`) still need spaces after their colons.

**Pull request merged**